DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests.cs

```diff
@@ -89,12 +89,12 @@ public void Registration_journey_has_no_accessibility_errors()
             lastName.SendKeys("User");
             var email = driver.FindElement(By.Id("Email"));
             email.SendKeys("candidate@test.com");
+            var centre = new SelectElement(driver.FindElement(By.Id("Centre")));
+            centre.SelectByValue("2");

```

Comment:
This test is now moved to a different file on master - should be an easy one to resolve.